starboard/nplb/socket_set_options_test.cc

```diff
@@ -11,6 +11,7 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
+#if SB_API_VERSION < 16
```

Comment:
Let's not do that yet. Once you remove  all the SbSocketSet all e.g SbSocketSetSendBufferSize from SB16 then we should turn off the test.